### PR TITLE
test: Move from CDP to BiDi

### PR DESCRIPTION
**Pull request by martinpitt**
See https://github.com/cockpit-project/cockpit/pull/20832

Explicitly add a "glob" devDependency to follow suit with
https://github.com/cockpit-project/cockpit/commit/680decc155a
It was previously used implicitly through a transient dependency of
something else, but our esbuild po-plugin uses it explicitly.

Adjust `waitDownloadFile()` to the new download directory attribute.
This is a `Path` object now, which is more convenient.

### CODE REVIEW

**Comment by martinpitt** [comment]
So, this looks quite okay. testUpload() cannot be run multiple times, but that feels like an unrelated test bug.

**Comment by martinpitt** [comment]
I fixed testUpload() in #701, and included it here.

**Comment by martinpitt** [comment]
Good, the rest are flakes and changed pixel refs.

File: package.json [hunk 1]
```diff
@@ -37,6 +36,7 @@
     "eslint-plugin-react": "7.35.0",
     "eslint-plugin-react-hooks": "4.6.2",
     "gettext-parser": "8.0.0",
+    "glob": "11.0.0",
```
Comment (allisonkarlitskaya) [inline package.json:39]: Huh.  Could have sworn I had already done that...

Probably in one of my many unmerged PRs ;)

**Review by allisonkarlitskaya — APPROVED** [review]
Enjoying the pathlib stuff :)

**Pull request merged**